photon-lib/src/main/java/org/photonvision/PhotonUtils.java

```diff
@@ -206,4 +213,50 @@ public static Rotation2d getYawToPose(Pose2d robotPose, Pose2d targetPose) {
     public static double getDistanceToPose(Pose2d robotPose, Pose2d targetPose) {
         return robotPose.getTranslation().getDistance(targetPose.getTranslation());
     }
+
+    /**
+     * Corrects a given pixel for perspective distortion
+     *
+     * @param pixel pixel to be corrected
+     * @param camIntrinsics the distortion matrix of the camera
+     * @return the angles
+     */
+    public static Rotation3d correctPixelRot(Point pixel, Matrix<N3, N3> camIntrinsics) {
```

Comment:
Maybe just a merge commit meme, but getCorrectedPixelRot already exists? Let's try to only implement this agorithm in only one place

Reply:
Unfortunately it's not an accident. I copied it from SimCameraProperties because the math worked for correcting perspective distortion, and I couldn't access the function without creating a SimCameraProperties object. Would it fit with PhotonVision convention to create a PhotonImageUtils class for that set of pixel functions?